src/fragments/lib/auth/native_common/getting_started/common.mdx

```diff
@@ -116,13 +152,13 @@ If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can
                                 "profile",
                                 "aws.cognito.signin.user.admin"
                             ]
-                        }
+                        },
```

Comment:
nit: this trailing comma make it invalid json string...

Reply:
Great catch, fixed with most recent commit.